add-on/chrome/skin/shared/loop.css

```diff
@@ -180,11 +180,16 @@
     color: #00a9dc;
   }
 
+  notification[value="loop-sharing-notification"] .notification-inner description {
+    padding-top: .2rem;
+    padding-bottom: .4rem;
+  }
+
   notification[value="loop-sharing-notification"] .notification-button {
     -moz-appearance: none;
     background-color: #fff;
     border: 0;
-    width: 100px;
+    padding: .2rem 1rem .4rem;
     height: 40px;
     margin: 0;
     list-style-image: url(chrome://loop/content/shared/img/pause-12x12.svg);
```

Comment:
You didn't touch this line, but do we actually want this list-style-image here? If not, we should remove it.